koku/api/utils.py

```diff
@@ -450,35 +450,60 @@ def materialized_view_month_start(dh=DateHelper()):
     return dh.this_month_start - relativedelta(months=settings.RETAIN_NUM_MONTHS - 1)
 
 
+def get_split_date_string(date_str):
+    """Return split date string"""
+
+    if not isinstance(date_str, str):
+        return date_str
+
+    split_at_char = ["T", " "]
+    for char in split_at_char:
+        if char.lower() in date_str.lower():
+            return date_str.split(char)[0]
+    return date_str
+
+
 def get_months_in_date_range(report=None, start=None, end=None, invoice_month=None):
     """returns the month periods in a given date range from report"""
+
     dh = DateHelper()
     if report:
-        if report.get("start") and report.get("end"):
-            LOG.info(f"using start: {report.get('start')} and end: {report.get('end')} dates from manifest")
-            start_date = report.get("start")
-            end_date = report.get("end")
-            if report.get("invoice_month"):
-                LOG.info(f"using invoice_month: {report.get('invoice_month')}")
-                invoice_month = report.get("invoice_month")
+        manifest_start = report.get("start")
+        manifest_end = report.get("end")
+        manifest_invoice_month = report.get("invoice_month")
+
+        if manifest_start and manifest_end:
+            LOG.info(f"using start: {manifest_start} and end: {manifest_end} dates from manifest")
+            start_date = manifest_start
+            end_date = manifest_end
+            if manifest_invoice_month:
+                LOG.info(f"using invoice_month: {manifest_invoice_month}")
+                invoice_month = manifest_invoice_month
         else:
             LOG.info("generating start and end dates for manifest")
             start_date = DateAccessor().today() - datetime.timedelta(days=2)
             start_date = start_date.strftime("%Y-%m-%d")
             end_date = DateAccessor().today().strftime("%Y-%m-%d")
```

Comment:
Initializing two different `DateAccessor` objects results in different timestamps since `timezone.now()` is called when the object is created. Practically speaking, it's most likely microseconds of difference. But these should use a single object for consistency and to avoid the overhead of creating two objects when one will do.

Reply:
Actually, there's already a `DateHelper` object. Just use that. Also, keep these as `datetime` objects.
```suggestion
            start_date = dh.today - datetime.timedelta(days=2)
            end_date = dh.today
```